docs/ManualClusterSetup.md

```diff
@@ -47,7 +52,33 @@ Get the calico binary onto each host.  You can download a specific [release](htt
 Finally, you can optionally preload the Calico Docker image.  This will make the Calico container creation more responsive the first time you run it.  Select the same version of the Calico Docker image as you selected above.  For example, to pull the latest released version
```

Comment:
I think it's worth tightening up the language here around the "calico docker image" and "calico container". We should introduce the calico-node term and describe it as our agent that runs on each compute host.